modules/pruning-images-manual.adoc

```diff
@@ -0,0 +1,325 @@
+  '{{range $ii, $item := $tag.items}}{{if eq $item.image "'"${image_name}"\
+  $'"}}{{$is.metadata.name}}:{{$tag.tag}} at position {{$ii}} out of {{len $tag.items}}\n'\
+  '{{end}}{{end}}{{end}}{{end}}'
+myapp:v2 at position 4 out of 5
+myapp:v2.1 at position 2 out of 2
+myapp:v2.1-may-2016 at position 0 out of 1
+----
+
+When default options are used, the image is never pruned because it occurs at
+position `0` in a history of `myapp:v2.1-may-2016` tag. For an image to be
+considered for pruning, the administrator must either:
+
+* Specify `--keep-tag-revisions=0` with the `oc adm prune images` command.
++
+[CAUTION]
+====
+This action effectively removes all the tags from all the namespaces with
+underlying images, unless they are younger or they are referenced by objects
+younger than the specified threshold.
+====
+
+* Delete all the `istags` where the position is below the revision threshold,
+which means `myapp:v2.1` and `myapp:v2.1-may-2016`.
+
+* Move the image further in the history, either by running new Builds pushing to
+the same `istag`, or by tagging other image. Unfortunately, this is not always
+desirable for old release tags.
+
+Tags having a date or time of a particular image's Build in their names should
+be avoided, unless the image must be preserved for an undefined amount of time.
+Such tags tend to have just one image in its history, which effectively prevents
+them from ever being pruned.
+
+[discrete]
+[id="pruning-images-secure-against-insecure_{context}"]
+==== Using a secure connection against insecure registry
+
+If you see a message similar to the following in the output of the `oc adm prune images`
+command, then your registry is not secured and the `oc adm prune images`
+client attempts to use a secure connection:
+
+----
+error: error communicating with registry: Get https://172.30.30.30:5000/healthz: http: server gave HTTP response to HTTPS client
+----
+
+. The recommend solution is to secure the registry. Otherwise, you can force the
+client to use an insecure connection by appending `--force-insecure`  to the
+command, however this is not recommended.
+
+[discrete]
+[id="pruning-images-insecure-against-secure_{context}"]
+==== Using an insecure connection against a secured registry
+
+If you see one of the following errors in the output of the `oc adm prune images`
+command, it means that your registry is secured using a certificate signed by a
+certificate authority other than the one used by `oc adm prune images` client for
+connection verification:
+
+----
+error: error communicating with registry: Get http://172.30.30.30:5000/healthz: malformed HTTP response "\x15\x03\x01\x00\x02\x02"
+error: error communicating with registry: [Get https://172.30.30.30:5000/healthz: x509: certificate signed by unknown authority, Get http://172.30.30.30:5000/healthz: malformed HTTP response "\x15\x03\x01\x00\x02\x02"]
+----
+
+By default, the certificate authority data stored in the user's configuration
```

Comment:
s/data stored in the user's configuration files are used/data stored in the user's configuration files is used/

From ISG:
> data n
> Use as a singular noun with a singular verb, even though “data” is the plural of “datum.”